src/App.js

```diff
@@ -1,23 +1,140 @@
+import LikeButton from './components/LikeButton/LikeButton';
+import Random from './components/Random/Random';
+import Rating from './components/Rating/Rating';
+import picture1 from './assets/images/maxence.png'
+import picture2 from './assets/images/maxence-glasses.png'
+
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <p className='m-5'><b>Iteration 1</b></p>
+      <div>
+        <IdCard
+          lastName='Doe'
+          firstName='John'
+          gender='male'
+          height={178}
+          birth={new Date("1992-07-14")}
+          picture="https://randomuser.me/api/portraits/men/44.jpg"
+        />
+
+        <IdCard
+          lastName='Delores '
+          firstName='Obrien'
+          gender='female'
+          height={172}
+          birth={new Date("1988-05-11")}
+          picture="https://randomuser.me/api/portraits/women/44.jpg"
+        />
+      </div>
+
+      <p className='m-5'><b>Iteration 2</b></p>
+      <div>
+      <Greetings lang="de" children="Ludwig"/>
+      <Greetings lang="fr" children="François"/>
+      </div>
+
+      <p className='m-5'><b>Iteration 3</b></p>
+      <div>
+        <Random min={1} max={6}/>
+        <Random min={1} max={100}/>
+      </div>
+
+      <p className='m-5'><b>Iteration 4</b></p>
+      <div className='d-flex flex-sm-column align-items-center'>
+        <BoxColor r={255} g={0} b={0} />
+        <BoxColor r={128} g={255} b={0} />
+      </div>
+
+      <p className='m-5'><b>Iteration 5</b></p>
+      <div className='d-flex justify-content-center'>
+        <CreditCard
+          type="Visa"
+          number="0123456789018845"
+          expirationMonth={3}
+          expirationYear={2021}
+          bank="BNP"
+          owner="Maxence Bouret"
+          bgColor="#11aa99"
+          color="white" 
+        />
+            
+        <CreditCard
+          type="Master Card"
+          number="0123456789010995"
+          expirationMonth={3}
+          expirationYear={2021}
+          bank="N26"
+          owner="Maxence Bouret"
+          bgColor="#eeeeee"
+          color="#222222"
+        />
+            
+        <CreditCard
+          type="Visa"
+          number="0123456789016984"
+          expirationMonth={12}
+          expirationYear={2019}
+          bank="Name of the Bank"
+          owner="Firstname Lastname"
+          bgColor="#ddbb55"
+          color="white" 
+        />
+      </div>
+
+      <p className='m-5'><b>Iteration 6</b></p>
+      <div className='d-flex flex-sm-column align-items-center'>
+        <Rating children={0}/>
```

Comment:
```suggestion
        <Rating>0</Rating>
```
Esto, lo puedes hacer como lo has hecho pero es un mal uso, para pasar los children se haria asi